Exception on /get_role_listing/1 [GET]
Traceback (most recent call last):
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    return self.finalize_request(rv)
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 889, in finalize_request
    response = self.make_response(rv)
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 1161, in make_response
    raise TypeError(
TypeError: The view function for 'get_role_listing' did not return a valid response. The function either returned None or ended without a return statement.
Exception on /get_role_listing/1 [GET]
Traceback (most recent call last):
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    return self.finalize_request(rv)
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 889, in finalize_request
    response = self.make_response(rv)
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 1161, in make_response
    raise TypeError(
TypeError: The view function for 'get_role_listing' did not return a valid response. The function either returned None or ended without a return statement.
Exception on /rolelistings_open [GET]
Traceback (most recent call last):
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/app.py", line 526, in get_all_open
    rolelistings = RoleListings.query.filter_by(role_listing_open=True).all()
AttributeError: 'list' object has no attribute 'all'
Exception on /rolelistings_open [GET]
Traceback (most recent call last):
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/app.py", line 527, in get_all_open
    if len(rolelistings):
TypeError: object of type 'Query' has no len()
Exception on /rolelistings_open [GET]
Traceback (most recent call last):
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/app.py", line 526, in get_all_open
    rolelistings = RoleListings.query.filter_by(role_listing_open=True).all()
AttributeError: 'list' object has no attribute 'all'
Exception on /rolelistings_open [GET]
Traceback (most recent call last):
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/app.py", line 526, in get_all_open
    rolelistings = RoleListings.query.filter_by(role_listing_open=True).all()
AttributeError: 'list' object has no attribute 'all'
Exception on /rolelistings_open [GET]
Traceback (most recent call last):
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask_cors/extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/envMac/lib/python3.10/site-packages/flask/app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/chiragnagpal/Documents/SMU/Y3-S1/Software Project Management/Project/spmg8t4/app.py", line 526, in get_all_open
    rolelistings = RoleListings.query.filter_by(role_listing_open=True).all()
AttributeError: 'list' object has no attribute 'all'
